src/course-unit/hooks.jsx

```diff
@@ -100,6 +101,10 @@ export const useCourseUnit = ({ courseId, blockId }) => {
     },
   };
 
+  const handleXBlockDragAndDrop = (xBlockListIds, restoreCallback) => {
+    dispatch(setXBlockOrderListQuery(blockId, xBlockListIds, restoreCallback));
```

Comment:
```suggestion
  const handleXBlockDragAndDrop = (xblockListIds, restoreCallback) => {
    dispatch(setXBlockOrderListQuery(blockId, xblockListIds, restoreCallback));
```

Reply:
Carmel Case is tricky, but I agree with Vlad we should do either the whole word in lowercase or both litters XB in uppercase

refs:
https://github.com/openedx/frontend-app-learning/blob/9dd79ca77ecb46dd0ea616945daa21d49fea5ee8/src/courseware/course/sequence/Unit/urls.js#L16
https://github.com/openedx/frontend-lib-content-components/blob/f33a3b5521d709ae4b0a98a5bdd9744060177816/addXblock.js#L5
https://github.com/openedx/edx-platform/blob/fa66728a67705723bd9cc74d3ccc9ca3bda4b7ae/cms/static/js/views/utils/xblock_utils.js#L184

Reply:
corrected, thanks guys